app/src/main/res/layout/item_trend.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content">
+
+    <TextView
+        android:id="@+id/tvIndex"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginLeft="10dp"
+        android:layout_marginTop="15dp"
+        tools:text="1. Trending" />
+
+    <TextView
+        android:id="@+id/tvTrendName"
+        android:layout_width="250dp"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/tvIndex"
+        android:layout_marginLeft="10dp"
+        android:text="TextView"
+        android:textSize="18sp"
+        android:textStyle="bold"
+        tools:text="#ChainedToTheRhythm" />
+
+    <Button
+        android:id="@+id/btExplore"
```

Comment:
Some of your buttons IDs are "bt" and some are "btn", so it might be better to choose one for consistency